bin/z-gerrit

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ -n "$Z_LIB" ]; then
+    . $Z_LIB/env.sh
+else
+    echo "ERROR Incorrectly configured environment, Z_LIB not set" >&2
+    exit 1
+fi
+
+. $Z_LIB/gerrit.functions
+
+usage="Usage: $(basename $0) [-d] [gerrit_command]
+  -h - this help
+  -d - debug
+  gerrit_command - the command to execute via the gerrit SSH interface
+
+Executes the given gerrit command. Details about supported commands
+can be found in the documentation:
+https://gerrit.googlecode.com/svn/documentation/2.0/cmd-index.html"
+
+while [ $# -gt 0 ]; do
+  case "$1" in
+    -h)
+      echo "$usage"
+      exit 0
+    ;;
+    -d)
+      set -x
+    ;;
+    *)
+      break
+    ;;
+  esac
+  shift
+done
+
+declare -r gerrit_host=$(z_gerrit_host)
+if [ -z "$gerrit_host" ]; then
+  echo "ERROR: Can't find gerrit host" >&2
+  exit 1
+fi
+
+gerrit_prefix="ssh -To BatchMode=yes ${gerrit_host} gerrit"
```

Comment:
Should be a function that accepts arguments and arguments are checked for sanity instead of an undeclared variable. :)